src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -147,6 +148,7 @@ public void addClient(Client client) {
     public void setClient(Client target, Client editedClient) {
         requireAllNonNull(target, editedClient);
         tbmManager.setClient(target, editedClient);
+        initialiseTagNoteMap();
```

Comment:
is there anything we can do in tagnotemap that would allow us to replace a client instead of doing re-init?